daphne_worker/src/dap.rs

```diff
@@ -74,8 +74,8 @@ impl<'a, D> HpkeDecrypter<'a> for DaphneWorkerConfig<D> {
 
     async fn get_hpke_config_for(
         &'a self,
-        _task_id: &Id,
-    ) -> std::result::Result<Option<GuardedHpkeReceiverConfig<'a>>, DapError> {
+        _task_id: Option<&Id>,
```

Comment:
It's not obvious (to me) that this is ignored. Can we document that?

Reply:
That what is ignored, the task ID? This is denoted by the "_" in front. This is a fairly common pattern for Rust, and you'll find it throughout the code base. IMO this is sufficiently "self-documenting".